peps/pep-0729.rst

```diff
@@ -2,12 +2,13 @@ PEP: 729
 Title: Typing governance process
 Author: Jelle Zijlstra <jelle.zijlstra@gmail.com>, Shantanu Jain <hauntsaninja at gmail.com>
 Discussions-To: https://discuss.python.org/t/pep-729-typing-governance-process/35362
-Status: Draft
+Status: Accepted
```

Comment:
```suggestion
Status: Active
```

Re: https://github.com/python/peps/pull/3540#pullrequestreview-1740785544

Should this Process PEP be active?

Reply:
Yes, that's probably better